libs/core/src/lib/utils/pipes/list-group.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'listGroupPipe'
+})
+
+export class ListGroupPipe implements PipeTransform {
+
+    transform(items: any[], group: Function): any {
```

Comment:
I dont know if its possible to use Pipe in HTML likt that, but it would be great if we have generic types for that function.